test/test_images/kafka-proxy/main.go

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
Can you add a README file within this folder for this test image? What it does, how to configure it, etc.

Only very briefly as it is not a deliverable of the Knative project

Reply:
good idea! Will do.